www/generate_tutorial/src/tutorial.roc

```diff
@@ -41,7 +41,6 @@ viewNavbar =
             a [class "home-link", href "/", title "The Roc Programming Language"] [text "roc"],
             div [id "top-bar-links"] [
                 a [href "/tutorial"] [text "tutorial"],
-                a [href "/install"] [text "install"],
```

Comment:
Maybe this should direct to https://www.roc-lang.org/tutorial#installation on the tutorial page instead, for now?

Reply:
Good point, https://www.roc-lang.org/tutorial#installation is like a thin wrapper around the github getting-started so I've linked it straight to there.